js/GoogleLanguageTranslation.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * Created by arjun on 5/22/2016.
+ */
+function googleTranslateElementInit()
```

Comment:
It is just google's js for language selection. It is not necessary to use though I have used it.  :)